test/templates/peervnet.json

```diff
@@ -0,0 +1,152 @@
+          {
+            "name": "ipconfig1",
+            "properties": {
+              "privateIPAllocationMethod": "Dynamic",
+              "publicIPAddress": {
+                "id": "[resourceId('Microsoft.Network/publicIPAddresses','publiciptest')]"
+              },
+              "subnet": {
+                "id": "[resourceId('Microsoft.Network/virtualNetworks/subnets', 'vnet', 'subnettest')]"
+              }
+            }
+          }
+        ]
+      }
+    },
+    {
+      "apiVersion": "2018-10-01",
+      "type": "Microsoft.Compute/virtualMachines",
+      "name": "test",
+      "location": "eastus",
+      "dependsOn": [
+        "[resourceId('Microsoft.Storage/storageAccounts/', concat(uniquestring(resourceGroup().id), 'vnettest'))]",
+        "[resourceId('Microsoft.Network/networkInterfaces/', 'nictest')]"
+      ],
+      "properties": {
+        "hardwareProfile": {
+          "vmSize": "Standard_A1"
+        },
+        "osProfile": {
+          "computerName": "test",
+          "adminUsername": "cloud-user",
+          "linuxConfiguration": {
+            "disablePasswordAuthentication": true,
+            "ssh": {
+              "publicKeys":[
+                {
+                  "keyData": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQCr86xjoaVhnTa28DGGKrqujt9XNppdQFxcuYqoeCpGXQLsQyrPULwK/P5uu6sdlbbw1CCiBVqQxjXZzG6iHhBOb0AUj4EilJ47+OXMWVURqPCMquX0gY18xq2xHEiYvUBw4F2q4eWIMkmv68Ljf4KJvLfRyP7UiUa4vgf9wyY3MjhM7XpSPIKTcqfkD/Uiy06KH0BE3FpeQ5rbpNh7vHI+Ir6DbCuLRqbmDyXuqIo/Av5WV8tABQe/XEVFjQgHrklX53kjMUGyctn2FmvmazWBVXPzq+BmIXlaW51tsNdbf/OsjEZW6c6WGm1eZdjfgQMYvXyJD+NPCT3lrNy06zjx",
+                  "path": "/home/cloud-user/.ssh/authorized_keys"
+                }
+              ]
+            }
+          }
+        },
+        "storageProfile": {
+          "imageReference": {
+            "publisher": "OpenLogic",
+            "offer": "CentOS",
+            "sku": "7.5",
+            "version": "latest"
+          },
+          "osDisk": {
+            "createOption": "FromImage"
+          },
+          "dataDisks": [
+            {
+              "diskSizeGB": 1023,
+              "lun": 0,
+              "createOption": "Empty"
+            }
+          ]
+        },
+        "networkProfile": {
+          "networkInterfaces": [
+            {
+              "id": "[resourceId('Microsoft.Network/networkInterfaces','nictest')]"
+            }
+          ]
+        },
+        "diagnosticsProfile": {
+          "bootDiagnostics": {
+            "enabled": true,
+            "storageUri": "[concat(reference(concat('Microsoft.Storage/storageAccounts/', concat(uniquestring(resourceGroup().id), 'vnettest')), '2016-01-01').primaryEndpoints.blob)]"
+          }
+        }
+      }
+    }
+  ],
+  "outputs": {
+    "hostname": {
+      "type": "string",
+      "value": "[reference('publiciptest').dnsSettings.fqdn]"
+    },
+    "vnetid": {
+      "type": "string",
+      "value": "[reference('vnet').resourceGuid]"
```

Comment:
This is not the value required for `peerVnetID`.  Will need to either return it or derive it above.